packages/envoyproxy/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,97 @@
+---
+description: Pipeline for normalizing envoyproxy logs
+processors:
+- set:
+    field: ecs.version
+    value: 8.11.0
+- set:
+    field: event.kind
+    value: event
+- append:
+    field: event.category
+    value: network
+- set:
+    field: event.ingested
+    value: '{{_ingest.timestamp}}'
+- pipeline:
+    if: ctx.message.charAt(0) == (char)("{")
```

I think using `if: ctx.message.startsWith('{')` is more cleaner.
```suggestion
    if: ctx.message.startsWith('{')
```